### PR TITLE
Dev/posts review

### PR DESCRIPTION
**Pull request by slashplusweb**
My changes as discussed. 

Moreover I optimized the markdown component a bit and added a markdown compiler as well as a markdown preview.

### CODE REVIEW

File: resources/js/components/Fields/MarkdownField.vue [hunk 1]
```diff
@@ -24,37 +45,52 @@
...
+        watch: {
+            data(data) {
+                this.update(data);
+            },
         },
-
         mounted() {
-            this.editor = CodeMirror.fromTextArea(document.getElementById('editor'), {
-                value: this.value || '',
+            let self = this;
```
Comment (slashplusweb) [inline resources/js/components/Fields/MarkdownField.vue]: error

File: src/Models/Post.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public function author()
 
     public function tags()
     {
-        return $this->belongsToMany(Tag::class);
+        return $this->belongsToMany(Tag::class)->withTimestamps();
```
Comment (slashplusweb) [inline src/Models/Post.php]: fixes 

intermediate-table

File: resources/js/components/Fields/MarkdownField.vue (hunk 1 above)
Comment (bambamboole) [reply resources/js/components/Fields/MarkdownField.vue]: i will check this out and try to reproduce.

Comment (bambamboole) [reply resources/js/components/Fields/MarkdownField.vue]: I have a solution and I will put it in another review comment

**Pull request merged**